src/index.ts

```diff
@@ -14,23 +15,25 @@ export type Lens<T, R> = {
 }
 export type Scope<T, R> = string | number | Lens<T, R>;
 
-export function pick(selector: Selector | string) {
+export function pick<T>(selector: Selector | string) {
```

Comment:
Maybe we can use here generic defaults https://github.com/Microsoft/TypeScript/pull/13487